src/ClientBuilder.php

```diff
@@ -334,8 +334,8 @@ private function createTransportInstance()
             return new NullTransport();
         }
 
-        if (!$this->messageFactory instanceof MessageFactory) {
-            throw new \RuntimeException('MessageFactory is not ready');
+        if (null === $this->messageFactory) {
+            throw new \RuntimeException('The PSR-7 message factory is not ready');
```

Comment:
Change this message to `The PSR-7 message factory must be set.` to be coherent with the rest of the code